### DISCUSSION TITLE
Trace Span Display "Missing"

### DISCUSSION THREAD

**Discussion by code-jar** [datalust/seq-tickets#2238] [Help] [answered]
Gateway requests for back-end services
![image](https://github.com/user-attachments/assets/fbd486a0-d8bb-479f-a143-eb810569c9a7)

**Comment by liammclennan** [comment, marked as answer]
Hi @code-jar 

Spans have a `@SpanId` and a `@ParentId`. Your `/api/account/save` span has a `@ParentId` of a span that has not been sent to Seq.

**Comment by code-jar** [comment]
I found the reason, the front-end service added HTTP tracking using “AddHttpClientInstrumentation” and I filtered out the HTTP 'Span' records. The missing 'spanId' is this one.
[HttpHandlerDiagnosticListener](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/HttpHandlerDiagnosticListener.cs#L599)
